src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -8,7 +8,7 @@
     "NuGet.Versioning": "3.6.0-rc-1987",
     "System.Collections.Immutable": "1.1.37",
     "System.Reflection.Metadata": "1.1.0",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
+    "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24806-04"
```

Comment:
Are you saying that not pulling in the closure might cause issues, or that packages pulled in by the closure might cause issues? I have confirmed that the only file that changes in the resulting buildtools package is `lib/net45/System.Runtime.InteropServices.RuntimeInformation.dll`, if that is expected. Since this upgrade is only on the net45 side (is that an insufficient fix?), I don't think there is any extended closure.